pkg/api/v1/types.go

```diff
@@ -710,6 +710,8 @@ type ISCSIVolumeSource struct {
 	IQN string `json:"iqn"`
 	// iSCSI target lun number.
 	Lun int `json:"lun"`
+	// Optional: Defaults to 'default' (tcp). iSCSI interface name that uses an iSCSI transport.
```

Comment:
The changes to the README already provides this information and more as well as links to configuration docs